logic/broadcaster.js

```diff
@@ -123,9 +116,8 @@ Broadcaster.prototype.getPeers = function (params, cb) {
 			return setImmediate(cb, err);
 		}
 
-		if (self.consensus !== undefined && originalLimit === constants.maxPeers) {
-			self.consensus = modules.peers.getConsensus(peers);
-			library.logger.info(['Broadhash consensus now', self.consensus, '%'].join(' '));
+		if (originalLimit === constants.maxPeers) {
```

Comment:
Imo there is no need to calculate consensus and log here, consensus is not useful during broadcasts.

Reply:
I've left the log but reused cached consensus.